core/src/test/java/tachyon/worker/hierarchy/CapacityUsageTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.hierarchy;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Properties;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.client.OutStream;
+import tachyon.client.TachyonFS;
+import tachyon.client.TachyonFile;
+import tachyon.client.WriteType;
+import tachyon.master.LocalTachyonCluster;
+import tachyon.util.CommonUtils;
+
+public class CapacityUsageTest {
+  private static final int MEM_CAPACITY_BYTES = 20 * Constants.MB;
+  private static final int DISK_CAPACITY_BYTES = Constants.GB;
+  private static final int USER_QUOTA_UNIT_BYTES = Constants.MB;
+  private static final int HEARTBEAT_INTERVAL_MS = 30;
+
+  private LocalTachyonCluster mLocalTachyonCluster = null;
+  private TachyonFS mTFS = null;
+
+  @After
+  public final void after() throws Exception {
+    mLocalTachyonCluster.stop();
+  }
+
+  @Before
+  public final void before() throws IOException {
+    Properties workerProps = new Properties();
+    workerProps.setProperty("tachyon.worker.hierarchystore.level.max", "2");
```

Comment:
This is not consistent with the way we setup unit test. Could you please take a look at how other tests setup customized properties?